Docs/source/Software/drivers.rst

```diff
@@ -16,10 +16,28 @@ Ethernet
 
 Purpose
 
+The Ethernet drivers are sending GPS, CAN, and IMU data to the telemetry team.
+The sunlight is the client while telemetry is the server
+
 Usage
 
+Ethernet_Init() is first called as a way to create the Queue and create a socket when a connection has been established.
+Ethernet_ConnectToServer() is a static blocking function called inside Ethernet_Init() that is either called when connection
+first needs to be established or when connection is lost with telemetry.  Ethernet_SendMessage() then collects data from the 
+Ethernet Queue and sends through Ethernet - if there is no connection, it calls Ethernet_ConnectToServer().Ethernet_SendMessage() returns pdFalse
+if the Queue is empty and pdTrue if the Queue is not empty. The struct EthernetMSG_t will hold the data packets of either information 
+regarding IMU, GPS, or CAN. The struct consists of an id enum, length, and a union of data. Since the data is in a union, there will be zero padding
+at the MSB.
+
```

Comment:
Markup all the function names. Also check to see if this compiles when you run 'make html'. Make sure everything you wrote looks the way you want it to when you run it.